README.md

```diff
@@ -1,6 +1,7 @@
 ![SendGrid Logo](https://uiux.s3.amazonaws.com/2016-logos/email-logo%402x.png)
 [![BuildStatus](https://api.travis-ci.org/sendgrid/smtpapi-php.png?branch=master)](https://travis-ci.org/sendgrid/smtpapi-php)
 [![Latest Stable Version](https://poser.pugx.org/sendgrid/smtpapi/version.png)](https://packagist.org/packages/sendgrid/smtpapi)
+[![Latest Version on Packagist](https://img.shields.io/packagist/v/sendgrid/sendgrid.svg?style=flat-square)](https://packagist.org/packages/sendgrid/sendgrid)
```

Comment:
Can you please link to https://packagist.org/packages/sendgrid/smtpapi rather than https://packagist.org/packages/sendgrid/sendgrid ?

Reply:
sorry, my bad
done